Dnn.CommunityMetrics.dnn

```diff
@@ -78,6 +78,10 @@
               <name>Dnn.CommunityMetrics.dll</name>
               <path>bin</path>
             </assembly>
+            <assembly>
+              <name>Octokit.dll</name>
```

Comment:
We should add the version in the manifest for binging redirects purpose

Reply:
Pushed a new commit.